app/modules/save_to_local.js

```diff
@@ -1,6 +1,14 @@
-import { tasks } from './taskArr.js';
-
 function saveToLocal(value) {
+  let tasks;
+  if (
+    localStorage.getItem('tasks') === null
+    || localStorage.getItem('tasks') === '[]'
+  ) {
+    tasks = [];
+  } else {
+    tasks = JSON.parse(localStorage.getItem('tasks'));
+  }
+
```

Comment:
- [ ] Kindly make sure when you edit the list, it is stored in the local storage. 😄

Reply:
Hi @jasonfelice, the code is actually updating the edited text on key Press === "Enter".

Reply:
Should I use another event listener?🙃